.github/workflows/tests.yml

```diff
@@ -1,14 +1,13 @@
 name: tests
 
-on:
-  pull_request:
+on: [push, pull_request]
```

Comment:
Add `push` and `pull_request` triggers on GitHub actions.

Reply:
I think we prefer it only run on `pull_request`; any reason you're adding `push`? Thanks!

Reply:
I plan to let my upcoming commits can be triggered by `push` on GitHub Action.

Anyway, I will revert this change :).